cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -34,7 +34,12 @@ <h1 class='m-text-introduction_heading'>{{ value.heading }}</h1>
         <p class='m-text-introduction_body'>{{ value.body | safe }}</p>
 
         <div class='m-text-introduction_link-container'>
-            <a class='jump-link' href='{{ value.link.url }}'>{{ value.link.text }}</a>
+            {% for link in value.links %}
+                <li class="list_item">
+                    <a class='jump-link' href='{{ link.url }}'>{{ link.text }}</a>
+                </li>
+            {% endfor %}
+
```

Comment:
@KimberlyMunoz hey is this good enough for now a list of links in text introduction?